releasenotes/notes/remove-ancilla-polynomial-rotation-62cad73f395270ca.yaml

```diff
@@ -0,0 +1,6 @@
+---
+other:
+  - |
+    Remove the use of an ancilla register in the polynomial pauli 
+    rotations and set the ancilla register to be None as default in 
+    QuantumCircuit.mcry().
```

Comment:
Can you make these two separate points? Like
```
- |
  The bit about PiecewisePoly
- |
  The bit about changing the behaviour of mcry
```

Reply:
changed in the code